### PR TITLE
tweaks and turns

### PR DESCRIPTION
**Pull request by robertavram**

### CODE REVIEW

**Comment by robertavram** [comment]
@adi130987 just realized.. partner should filter the agreements.. that's the only role of partner here..
I'll make the change

File: pmp/app-elements/agreements/agreement-data/agreement-list-data.html [hunk 1]
```diff
@@ -55,6 +60,19 @@
             value: null
           }
         },
+        actions: {
+          setAgreements: function(agreements) {
+            return {
+              type: 'SET_AGREEMENTS',
```
Comment (adi130987) [inline pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: Do we need the entire agreement data to be global? If we are gonna use it for esmm dropdown, then we only need the id and the agreement name.

File: pmp/app-elements/interventions/intervention-details.html [hunk 2]
```diff
@@ -33,18 +34,26 @@
...
-                                                   options="[[partnersDropdownData]]">
-            </etools-searchable-multiselection-menu>
+            <paper-input label="Partner Name"
+                           value="[[intervention.agreement_obj.partner_name]]"
+                           placeholder="&#8212;"
+                           readonly></paper-input>
           </div>
           <div class="col col-3">
-            <etools-searchable-multiselection-menu id="agreement"
+            <object-searchable-multi-selection id="agreements"
```
Comment (adi130987) [inline pmp/app-elements/interventions/intervention-details.html]: I think the purpose of `object-searchable-multi-selection` is to be able to supply any objects to the `esmm dropdown` and specify the value and label properties to be used. In this case we can use an array from redux store, similar with `partnersDropdownData`, that has each obj mapped

```javascript
{
value: agreement.id,
label: agreement_number
}
```

and when the selected value changes esmm will fire `value-change` that can be something like this:

```javascript
_interventionAgreementChanged: function(event) {
           this.set('intervention.agreement', event.detail.selectedValues.value);
}
```

File: pmp/app-elements/partners/partners-data/partners-list-data.html [hunk 3]
```diff
@@ -64,9 +70,37 @@
...
+        actions: {
+          setPartnersDropdown: function(partnersDropdownData) {
+            return {
+              type: 'SET_PARTNERS_DROPDOWN',
+              partnersDropdownData: partnersDropdownData
+            };
+          },
+          setPartners: function(partnersData) {
+            return {
+              type: 'SET_PARTNERS',
```
Comment (adi130987) [inline pmp/app-elements/partners/partners-data/partners-list-data.html]: Do we need all partners to be global?

**Comment by robertavram** [comment]
@adi130987 -> tired day.. I went through a bunch of different versions of selecting elements... we'll stick to what we pass in... we need the partners dropdown global. I'm not liking the event management and setting the id for every dropdown... but I guess it's ok for now... I was also looking in the future when we might have validation on some of those fields: for ex: Intervention to-date cannot be greater than agreement to-date, etc.. for that we need more of the agreement to come through not just the id and the value....

Today I'll fix this PR... the Partners are needed to filter the agreements if partner-agreement is selectable. I'll create one element for this field...

**Comment by robertavram** [comment]
@adi.. ok.. check now.. styling tweaks needed.. but that's basically what we need.. filter agreements by partner and choose agreement... on intervention detail page.. we shoul;d only worry about the agreement id , until we implement validation.. when we can just as easily expose the object up

**Comment by adi** [comment]
Hi, @adi here. Any reason you keep calling me to this thread?

**Comment by robertavram** [comment]
sorry @adi wrong Adi, github auto-correct

**Pull request merged**